src/renderer/base/renderer.cpp

```diff
@@ -326,40 +327,43 @@ DWORD Renderer::_timerToMillis(TimerRepr t) noexcept
 // - HRESULT S_OK, GDI error, Safe Math error, or state/argument errors.
 [[nodiscard]] HRESULT Renderer::PaintFrame()
 {
-    auto tries = maxRetriesForRenderEngine;
-    while (tries > 0)
+    HRESULT hr{ S_FALSE };
+    for (auto attempt = 0; attempt < maxRetriesForRenderEngine; ++attempt)

```

Comment:
This probably complains in x86 because attempt is int and maxRetriesForRenderEngine is unsigned int.